docs/appkit/overview.mdx

```diff
@@ -104,13 +112,50 @@ With a layered feature stack spanning onboarding to transactions, messaging and
 
 ![AppKit banner](../../static/img/appkit.png)
 
-Some of the key features of the AppKit SDK include:
+#### Framework Agnostic
+Integrate effortlessly with popular frameworks like React, Vue, Svelte, vanilla JavaScript, and more.
+
+#### EIP-6963 Support
+Seamlessly connect browser extension wallets for an enhanced user experience. Learn more about **EIP-6963 [here](https://eips.ethereum.org/EIPS/eip-6963).**
+
+#### Email Wallets
```

Comment:
you are repeating the some emails here

#### Email & Social Login
Expand your app’s reach by enabling users to connect using their email or social accounts. Implement these features with minimal code and leverage Smart Account functionality for advanced capabilities.